src/main/java/org/madhuri/app/message/dao/ChannelDAO.java

```diff
@@ -57,4 +57,31 @@ public void deleteChannel(long id) {
 			session.close();
 		}
 	}
+
+	public Channel getChannelById(long id) {
+		Session session = HibernateUtil.getSession();
+        Channel channel = session.get(Channel.class, id);
+        session.close();
+        return channel;
+	}
+
+	public Channel updateChannel(Channel existingChannel) {
```

Comment:
While updating, are we not populating updated_at time to specify the updated time ?